gcloud/datastore/__init__.py

```diff
@@ -111,43 +104,16 @@ def get_connection():
     return Connection(credentials=scoped_credentials)
 
 
-def get_dataset(dataset_id):
-    """Establish a connection to a particular dataset in the Cloud Datastore.
-
-    This is a shortcut method for creating a connection and using it
-    to connect to a dataset.
-
-    You'll generally use this as the first call to working with the API:
-
-    >>> from gcloud import datastore
-    >>> dataset = datastore.get_dataset('dataset-id')
-    >>> # Now you can do things with the dataset.
-    >>> dataset.query().kind('TestKind').fetch()
-    [...]
-
-    :type dataset_id: string
-    :param dataset_id: The id of the dataset you want to use.
-                       This is akin to a database name
-                       and is usually the same as your Cloud Datastore project
-                       name.
-
-    :rtype: :class:`gcloud.datastore.dataset.Dataset`
-    :returns: A dataset with a connection using the provided credentials.
-    """
-    connection = get_connection()
-    return Dataset(dataset_id, connection=connection)
-
-
-def _require_dataset():
-    """Convenience method to ensure DATASET is set.
+def _require_dataset_id():
```

Comment:
Cf ad78f59941219a7e5103873676fe6409a7b4a34f (passes the value in, tests explicitly against `None`).  Same for `_require_connection`.